interface/main/finder/dynamic_finder.php

```diff
@@ -104,7 +102,45 @@
  });
 
  // OnClick handler for the rows
- $('#pt_table tbody tr').live('click', function () {
+
+  $(document).on('keyup', '#textID', function (e) {
+	 var code = e.which;
+
+	 if(code == 13){
+		 
+		  // ID of a row element is pid_{value}
+		  var newpid = $('#pt_table tbody tr').attr('id').substr(4);
+		  // If the pid is invalid, then don't attempt to set 
```

Comment:
Although comments/documentation in code should always be welcomed, you could leverage php to prevent server sending comments to browser.